wrappers/wrapper.rb

```diff
@@ -1572,18 +1578,22 @@ def simplify_service_setup_duration_and_vehicle_setup_modifiers(vrp, result = ni
           additional_setup = vehicle[:simplified_additional_setup].to_i
 
           total_travel_time_correction = 0
+          previous_point_id = nil
           route[:activities].each{ |activity|
-            next if activity[:service_id].nil? || activity[:travel_time].to_i.zero?
+            next if activity[:point_id].nil? || previous_point_id == activity[:point_id]
+
+            previous_point_id = activity[:point_id]
+            next if activity[:service_id].nil?
 
             setup_duration = services_grouped_by_point_id[activity[:point_id]].first.activity[:simplified_setup_duration].to_i
 
             next if setup_duration.zero?
 
             time_correction = coef_setup * setup_duration + additional_setup
 
-            total_travel_time_correction += time_correction
+            total_travel_time_correction += time_correction.round
```

Comment:
We need to round only once at the end, otherwise we accumulate rounding errors and totals do not represent the reality.